auth-api/tests/docker/docker-cmpose-mac.yml

```diff
@@ -1,81 +0,0 @@
-version: "3"
-
-services:
-  keycloak:
```

Comment:
Jia would use this because the other docker compose wasn't working on her computer, should be ok though because I have a newer version in my PR

Reply:
@Jxio Does the docker container in my PR work for you? https://github.com/bcgov/sbc-auth/pull/3327

Reply:
fyi, the remaining docker compose from this PR worked on my mac

Reply:
yes, I tested, it working